packages/cardano-services/test/StakePool/HttpStakePoolMetadataService/HttpMetadataService.test.ts

```diff
@@ -202,23 +203,19 @@ describe('StakePoolMetadataService', () => {
     it('returns StakePoolMetadataServiceError with FailedToFetchExtendedSignature error code when it cant fetch the signature', async () => {
       let metadata: any;
 
-      let alreadyCalled = false;
-      const handler = async () => {
-        if (alreadyCalled) return { body: cip6ExtMetadataMock, code: 200 };
-        alreadyCalled = true;
+      const handler = async (req?: IncomingMessage) => {
+        if (req?.url === '/cip-6') return { body: cip6ExtMetadataMock, code: 200 };
+        if (req?.url === '/metadata') return { body: metadata, code: 200 };
 
-        return {
-          body: metadata,
-          code: 200
-        };
+        return { body: 'not found', code: 404 };
       };
 
       ({ closeMock, serverUrl } = await mockPoolExtMetadataServer(handler));
 
       metadata = {
         ...mainExtMetadataMock,
         extDataUrl: `${serverUrl}/${ExtMetadataFormat.CIP6}`,
-        extSigUrl: UNFETCHABLE,
```

Comment:
Is this variable still needed? If yes, maybe update other tests that use it as well?

Reply:
It is used in other tests which do not perform the fetch.
I tried to do that but that would increase the test complexity as now a pre computed hash can be used (the URL is always the same), changing it requires computing the hash on some more tests... I think this doesn't worth.

Reply:
Alright, let's keep it